app/talents/my-profile/page.tsx

```diff
@@ -1,240 +1,15 @@
-"use client";
+import MyProfile from "@/app/components/profile-form";
```

Comment:
import MyProfile from "@/app/components/my-profile-page";

To know it's a "component page".

Reply:
?